src/main/java/org/dependencytrack/tasks/BomUploadProcessingTask.java

```diff
@@ -158,6 +164,19 @@ public void inform(final Event e) {
 
     private void processBom(final Context ctx, final File bomFile) throws BomProcessingException {
         LOGGER.info("Consuming uploaded BOM (%s)".formatted(ctx));
+        WorkflowState bomConsumptionState = null;
+        List<Component> components = null;
+        List<ServiceComponent> services = null;
+        try (final var qm = new QueryManager()) {
+            WorkflowState consumptionState = qm.getWorkflowStateByTokenAndStep(ctx.uploadToken, WorkflowStep.BOM_CONSUMPTION);
+            if (consumptionState != null) {
+                consumptionState.setStartedAt(Date.from(Instant.now()));
+                bomConsumptionState = qm.persist(consumptionState);
+            } else {
+                LOGGER.warn("Workflow state not found in database so cannot be updated");
```

Comment:
Including `ctx` in this log would be helpful for correlation.

We should also add a `TODO` here to change this to `error`, and potential throw an exception once the workflow has been migrated completely. Eventually, `BomUploadProcessingTask` should never encounter an event for which no workflow exists.